recipe/build.sh

```diff
@@ -1,20 +1,12 @@
-mkdir -p build
-cd build
-
-if [ `uname` = "Darwin" ]; then
-      cpp_std=14
-else
-      cpp_std=17
-      sed -i 's#/home/conda/feedstock_root/build_artifacts/vtk_.*_build_env/x86_64-conda_cos6-linux-gnu/sysroot/usr/lib.*;##g' ${PREFIX}/lib/cmake/vtk-8.2/Modules/vtkhdf5.cmake 
-      sed -i 's#/home/conda/feedstock_root/build_artifacts/vtk_.*_build_env/x86_64-conda_cos6-linux-gnu/sysroot/usr/lib.*;##g' ${PREFIX}/lib/cmake/vtk-8.2/VTKTargets-release.cmake
-fi
+git clone --recurse-submodules https://github.com/trelau/SMESH.git
+cd SMESH
+python prepare.py
 
 cmake -G "Ninja" \
       -D CMAKE_BUILD_TYPE:STRING="Release" \
       -D CMAKE_INSTALL_PREFIX:FILEPATH=$PREFIX \
       -D CMAKE_PREFIX_PATH:FILEPATH=$PREFIX \
       -D Boost_NO_BOOST_CMAKE:BOOL=ON \
-      -D CMAKE_CXX_STANDARD=${cpp_std} \
```

Comment:
Might need to keep `-D CMAKE_CXX_STANDARD=14` to get the `throw` errors fixed on linux?